aconfigure.ac

```diff
@@ -2480,10 +2489,12 @@ AC_ARG_ENABLE(libwebrtc,
 			;;
 		     *win32* | *w32* | *darwin* | *linux*)
                          case $target in
-                             armv7l*gnueabihf)
-                                 ac_webrtc_instset=neon
-                                 ac_webrtc_cflags="-DWEBRTC_ARCH_ARM_V7 -mfloat-abi=hard -mfpu=neon"
-                                 ;;
+                             armv*gnueabihf)
+							 	if test "x$ax_cv_support_neon_ext" = "xyes"; then
+									ac_webrtc_instset=neon
+									ac_webrtc_cflags="-DWEBRTC_ARCH_ARM_V7 -mfloat-abi=hard -mfpu=neon"
+								fi
```

Comment:
Do we need an "else" here?
Otherwise `ac_webrtc_instset` will be unset.

Reply:
What do you propose for "else"? Empty variable?

Reply:
Either ac_webrtc_instset=sse2 or generic.
I'm not sure what it's supposed to be if neon is not supported in ARM machine.

Reply:
> Either ac_webrtc_instset=sse2


it was sse2 initially (before this patch), but arm devices do not support sse2 and compilation was failing due to that.

Reply:
If this is the case, then perhaps we don't need the neon detection above? (since there is no failover if it doesn't support neon).

Just modify `armv7l*gnueabihf` to `armv*gnueabihf`, wouldn't this suffice?

Reply:
> Just modify armv7l*gnueabihf to armv*gnueabihf, wouldn't this suffice?

That was my initial approach, but I've talked to colleagues and they proposed checking compiler flags, too.

Also, there's a typo in `armv*gnueabihf` - `v` is also not needed, I'm on my way with updates.

Reply:
Also, armv6 does not have neon support, AFAIK. So, let's stick to that check.

Reply:
added fallback to `generic`